context/etcd-mgmt.go

```diff
@@ -0,0 +1,36 @@
+package context
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func StartEtcd() {
+
+	log.Info("Starting etcd")
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Could not able to get hostname")
+	}
+
+	ip_2379 := strings.Join([]string{"http://", ":2379"}, hostname)
+
+	ip_2380 := strings.Join([]string{"http://", ":2380"}, hostname)
+
+	default_etcd := strings.Join([]string{"default=", ""}, ip_2380)
```

Comment:
Why do you need to construct default_etcd like this? You can still pass the default parameters like what you have done for other arguments.